shared/components/Talks/index.js

```diff
@@ -14,9 +25,8 @@ const Talks = ({ talks }) => {
           const speakerProps = pathOr({}, ['speakers', 0], talk);
           return (
             <div className="Talk" key={index}>
-              <Speaker {...speakerProps} />
-              <h4 className="Speaker__title">{talk.title}</h4>
-              <p className="Speaker__summary">{talk.summary}</p>
+              <Speaker {...speakerProps} collapsed={collapsed} />
+              {talkDetails(talk, collapsed)}
```

Comment:
why don't you do `!collapsed && <TalkDetails talk={talk}>`. I think its a bit weird to have functions that return markup but aren't components... but just my opinion, doesn't really matter.